setup.py

```diff
@@ -67,6 +67,7 @@ def main():
         namespace_packages=["google", "google.cloud"],
         install_requires=dependencies,
         extras_require={},
+        python_requires=">=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*,!=3.5.*",
```

Comment:
I notice this is the same way it's done in the doc you reference, but when I read it there, I was wondering why not go with a bit more succinct:

python_requires="==2.7.*,>=3.6"

Reply:
That spelling might work (my memory of the syntax is fuzzy), but it by eyeballs, it like the empty set to me. :)  "All versions which are exactly 2.7 AND GTE 3.6."

Reply:
Yeah, my hastily constructed mental model is probably not how it actually works, now that I think about it. It would have to use AND or OR logic. Ignore me!